Updated band details for Sentinel5P with reference to openEO hub collections (#216)
eu-cdse/documentation: APIs/OpenSearch.qmd

:::

> Eeach of these ordering can be be accompanied by:

Reviewer: ```suggestion
Each of these ordering can be be accompanied by:
```

Some additional parameters are strictly satellite-specific, e.g. polarisation, which is defined only for Sentinel-1.

> For every satellite (collection) its set of query-able parameters may be obtained by a query like:

Reviewer: this looks very weird, I guess this is easier:

```suggestion
The set of query-able parameters of a given satellite (collection) can be obtained with:
```
